pkg/scheduler/cache/cache.go

```diff
@@ -82,7 +82,7 @@ func newSchedulerCache(ttl, period time.Duration, stop <-chan struct{}) *schedul
 	}
 }
 
-// Snapshot takes a snapshot of the current schedulerCache. The method has performance impact,
```

Comment:
This one refers to the type and not the package. Let's leave it. (but the type's name is also redundant...)

Reply:
DONE!